recipe/meta.yaml

```diff
@@ -134,26 +130,7 @@ outputs:
     requirements:
       run:
         - {{ pin_subpackage('kernel-headers_' ~ cross_target_platform, exact=True) }}
-    test:
-      commands:
-        - test -f $PREFIX/{{ target_machine }}-conda-linux-gnu/sysroot/lib/libc.so.6
-        - test -f $PREFIX/{{ target_machine }}-conda-linux-gnu/sysroot/sbin/ldconfig
-        - test -f $PREFIX/{{ target_machine }}-conda-linux-gnu/sysroot/usr/lib/crt1.o
-        - test -f $PREFIX/{{ target_machine }}-conda-linux-gnu/sysroot/usr/include/limits.h
-        - test -f $PREFIX/{{ target_machine }}-conda-linux-gnu/sysroot/usr/include/gnu/stubs-64.h  # [cross_target_platform == "linux-64"]
-        - test -f $PREFIX/{{ target_machine }}-conda-linux-gnu/sysroot/usr/include/gnu/stubs.h  # [cross_target_platform != "linux-64"]
-        - test -d $PREFIX/{{ target_machine }}-conda-linux-gnu/sysroot/usr/share/locale
-        - test -f $PREFIX/{{ target_machine }}-conda-linux-gnu/sysroot/usr/bin/ldd
-        - test -f $PREFIX/{{ target_machine }}-{{ ctng_vendor }}-linux-gnu/sysroot/lib/libc.so.6
-        - find "${PREFIX}" \( -name 'libnsl*' -o -path '*/rpcsvc/yp*' \) | { ! grep . ; }
-        - find "${PREFIX}" \( -name 'libcrypt*' -o -name 'crypt.*' \) | { ! grep . ; }
```

Comment:
We still want these tests.

Reply:
Huh? We can't test an output that doesn't exist. But those same tests (AFAICT) still exist for the other output.

Reply:
Ah, I see what happened. I deleted only the second output with its tests, but git came up with a diff that ended up deleting the tests of the previous output.

Reply:
If you look at only https://github.com/conda-forge/linux-sysroot-feedstock/pull/78/commits/659b8978eb31de066add8d68e290bbc8b70c227d, you'll see what I mean (this kind of thing is why I try to come up with meaningful commits, because often the full diff of a PR can be confusing or downright useless).